src/include/iscsi-client/helps.rb

```diff
@@ -141,11 +143,8 @@ def initialize_iscsi_client_helps(include_target)
         _("<h1>Authentication</h1>") +
         _(
           "<p>The default setting here is <i>No Authentication</i>. Uncheck the checkbox if " \
-          "authentication is needed for security reasons. Enter <b>Username</b> and <b>Password</b> " \
-          "for 'Incoming Authentication', 'Outgoing Authentication' or for both.</p>" \
-          "<p><b>Please note:</b><br>" \
-          "'Incoming Authentication' here correlates to 'Outgoing Authentication' "\
-          "on iSCSI target server side and the other way round.</p>"
+          "authentication is needed for security reasons. Use <b>Authentication by Targets</b> or <b>Authentication by Initiator</b>" \
```

Comment:
The help texts look quite similar, could you share some parts to avoid double translation work for the translators? (Put the shared parts into a variable or a constant or define a method returning the shared string.)